roles/openstack_helm_endpoints/vars/main.yml

```diff
@@ -299,6 +299,35 @@ _openstack_helm_endpoints_dns:
       designate:
         password: "{{ openstack_helm_endpoints_designate_rabbitmq_password }}"
 
+_openstack_helm_endpoints_mdns:
+  name: minidns
+  hosts:
+    default: minidns
+    public: designate-mdns
+  host_fqdn_override:
+    default: "{{ openstack_helm_endpoints_designate_api_host }}"
+  path:
+    default: /
+  scheme:
+    default: 'tcp'
+  port:
+    ipc:
+      default: 5354
+
+_openstack_helm_endpoints_powerdns:
+  auth:
+    service:
+      token: chiave_segreta
```

Comment:
Shouldn't this be a secret value?

Reply:
removed `_openstack_helm_endpoints_powerdns` because we don't install powerdns using osh for now.